### PR TITLE
Feat  add ph documentation

### PR DESCRIPTION
**Pull request by renzothenoob**
## Changes

1. Translate contributors.json to fil
2. Translate docs.json to fil
3. Translate examples.json to fil
4. Translate global.json to fil
5. Translate home.json to fil
6. Translate resources.json to fil
7. Translate media.json to fil
8. Translate tutorial.json to fil
9. Add fil in lang

### CODE REVIEW

**Comment by mackignacio** [comment]
Nice @renzothenoob adjusting some of the non-colloquial words.

**Comment by davedbase** [comment]
Hi folks, this looks good. We should change the folder code to `ph`.

**Comment by mackignacio** [comment]
@davedbase What is `tl` mean?

**Comment by davedbase** [comment]
Hmmm, that's the ISO code for tagalog but I just noticed that Filipino is marked FIL (which is what you have).

Screen Shot 2021-11-27 at 9 25 35 AM

I'm really unfamiliar with this. Is the documentation written in a dialect of Tagalog or proper and does it matter for regional considerations?

**Comment by renzothenoob** [comment]
I renamed the folder to `fl` short for `fil` or Filipino.

**Comment by mackignacio** [comment]
Tagalog is a native dialect. The official language of the Philippines is `Filipino`.

**Comment by davedbase** [comment]
Ahhhhh. Sorry for my confusion! If everyone is satisfied with this I'd be happy to merge. If you have an interest the docs and tutorials could use translation but that's a HUGE job.

**Comment by mackignacio** [comment]
LGTM

**Comment by davedbase** [comment]
Fantastic. The last thing I need is for the language to be registered in the root list: https://github.com/solidjs/solid-site/blob/master/src/App.data.tsx#L6-L19

Once you do that we can merge and deploy =)

File: src/App.data.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ const langs: { [lang: string]: any } = {
   ru: async () => (await import('../lang/ru/ru')).default(),
   fa: async () => (await import('../lang/fa/fa')).default(),
   tr: async () => (await import('../lang/tr/tr')).default(),
+  fil: async () => (await import('../lang/fl/fil')).default(),
```
Comment (mackignacio) [inline src/App.data.tsx:18]: Change `fil` to `fl` to make it more consistent.

**Review by mackignacio — CHANGES_REQUESTED** [review]

**Comment by davedbase** [comment]
Thanks folks!

**Pull request merged**